From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Thu, 5 Sep 2013 01:19:12 +0100
Subject: autoconf toe step

Stop stepping on the toes of standard autotools variables: remove
special-case code to recognize and set variables that work fine, or
have standard variables with different names that work fine by
default.
---
 configure.in | 40 ----------------------------------------
 1 file changed, 40 deletions(-)

diff --git a/configure.in b/configure.in
index a667d94..f48aea8 100644
--- a/configure.in
+++ b/configure.in
@@ -88,46 +88,6 @@ AC_CHECK_SIZEOF(unsigned char, 1)
 dnl Checks for optional libraries and tools
 
 AM_WITH_DMALLOC
-AM_MAINTAINER_MODE
-
-AC_ARG_WITH(CC,
-[  --with-CC               choose compiler],
-  if test "$withval" != "no" && test "$withval" != "yes" ; then
-    CC="$withval"
-  fi
-)
-
-AC_ARG_WITH(CFLAGS,
-[  --with-CFLAGS           set C compiler flags],
-  if test "$withval" != "no" && test "$withval" != "yes" ; then
-    CFLAGS="$withval"
-  fi
-)
-
-AC_ARG_WITH(CPPFLAGS,
-[  --with-CPPFLAGS         set C preprocessor flags],
-  if test "$withval" != "no" && test "$withval" != "yes" ; then
-    CPPFLAGS="$withval"
-  fi
-)
-
-AC_ARG_WITH(LDFLAGS,
-[  --with-LDFLAGS          set linker flags],
-  if test "$withval" != "no" && test "$withval" != "yes" ; then
-    LDFLAGS="$withval"
-  fi
-)
-
-if test x$ac_cv_prog_gcc = xyes; then
-        if test x$opt_maintainer_mode = xyes; then
-          CFLAGS=" -ggdb3 -O2 -Wall -Wpointer-arith"
-        fi
-
-        if test x$opt_dmalloc_mode = xyes; then
-          AC_CHECK_LIB( dmalloc, main)
-          AC_DEFINE(USE_DMALLOC, 1, [dmalloc])
-        fi
-fi
 
 AC_ARG_ENABLE(md5,
 [  --disable-md5           disable the MD5 algorithm],
